warehouse/query-core/src/main/java/datawave/query/jexl/visitors/BoundedRangeIndexExpansionVisitor.java

```diff
@@ -105,7 +103,7 @@ protected void rebuildFutureJexlNode(FutureJexlNode futureJexlNode) {
         JexlNode currentNode = futureJexlNode.getOrigNode();
         IndexLookupMap fieldsToTerms = futureJexlNode.getLookup().lookup();
         
-        futureJexlNode.setRebuiltNode(JexlNodeFactory.createNodeTreeFromFieldsToValues(JexlNodeFactory.ContainerType.OR_NODE, new ASTEQNode(
-                        ParserTreeConstants.JJTEQNODE), currentNode, fieldsToTerms, expandFields, expandValues, futureJexlNode.isKeepOriginalNode()));
+        futureJexlNode.setRebuiltNode(JexlNodeFactory.createNodeTreeFromFieldsToValues(JexlNodeFactory.ContainerType.OR_NODE,
+                        futureJexlNode.isKeepOriginalNode(), currentNode, fieldsToTerms, expandFields, expandValues, futureJexlNode.isKeepOriginalNode()));
```

Comment:
I think the first instance of futureJexlNode.isKeepOriginalNode() should simply be a "true".  That is supposed to be the isNegated flag.